### PR TITLE
New check 7151 DynamoDB tables point-in-time recovery (PITR) enabled @ManuelUgarte

### PR DESCRIPTION
**Pull request by ManuelUgarte**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
This Pull request has a check called [extra7151]. It checks if a DynamoDB Table has the backup option point-in-time recovery enabled.

### CODE REVIEW

File: checks/check_extra7151 [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+CHECK_ID_extra7151="7.151"
+CHECK_TITLE_extra7151="[extra7151] Check if DynamoDB tables point-in-time recovery (PITR) is enabled."
+CHECK_SCORED_extra7151="NOT_SCORED"
+CHECK_TYPE_extra7151="EXTRA"
+CHECK_SEVERITY_extra7151="Medium"
+CHECK_ASFF_RESOURCE_TYPE_extra7151="AwsDynamoDbTable"
+CHECK_ALTERNATE_check7151="extra7151"
+CHECK_SERVICENAME_extra7151="dynamodb"
+CHECK_RISK_extra7151='If the DynamoDB Table does not have point-in-time recovery enabled, it is vulnerable to accidental write or delete operations.'
```
Comment (toniblyx) [inline checks/check_extra7151]: Variables with comma would break CSV, please use `;`.

**Review by w0rmr1d3r — APPROVED** [review]
Looks good to me!

**Comment by toniblyx** [comment]
Variables with comma would break CSV, please use ; instead.

**Comment by ManuelUgarte** [comment]
Thank you for the feedback! I have narrowed down the result with jq to only give ENABLED as a result. There are no commas in the variable now.

File: checks/check_extra7151 (hunk 1 above)
Comment (toniblyx) [reply checks/check_extra7151]: Thanks

**Review by toniblyx — APPROVED** [review]

**Pull request merged**